opsdroid/events.py

```diff
@@ -286,3 +288,35 @@ class Image(File):
         """Return the image dimensions `(w,h)`."""
         fbytes = await self.get_file_bytes()
         return get_image_size_from_bytesio(io.BytesIO(fbytes), len(fbytes))
+
+
+class RoomCreation(Event):
+    """Event class to represent the creation of a new room."""
+
+    def __init__(self, public=True, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.is_public = public
+
+
+class SetRoomAlias(Event):
```

Comment:
maybe use "address" or something a little more non-matrixy here?

Reply:
Yeah, wasn't sure what the more generic term would be. But I also wanted to avoid ambiguity in what this event actually does, if you're in matrix-only land. Possibly this could be a use-case for connector-specific events in core?